### PR TITLE
News section on home page

### PR DESCRIPTION
**Pull request by AymanReh**
Added 4 display cards to the home view, Right now only shows example data but when an API is found will replaced it. Contains a title, description, tags and image. Also contains a button which says "show more" when pressed 4 more cards will appear below the current ones

### CODE REVIEW

**Comment by LiamSingh64** [comment]
Is it worth making the Cards it's own component? It could use `v-for` to generate the card-body and just pass in an array of objects which are API call responses

**Comment by LiamSingh64** [comment]
Or should we just merge for now so there is a proof-of-concept on the site?

**Comment by advweb-grp1** [comment]
Remove the prefix for each section like "title". Remove the hardcoded API key in code. Change that so it imports it from the .env.local file instead.

**Comment by rwx-yxu** [comment]
current view:
image

File: src/views/HomeView.vue [hunk 1]
```diff
@@ -41,7 +41,78 @@
...
+                {{ card.title }}
+              </h5>
+              <p class="card-text">
+                {{ card.description }}
+              </p>
+              <p class="card-text">
+                {{ card.date }}
+              </p>
+            </div>
+          </div>
```
Comment (rwx-yxu) [inline src/views/HomeView.vue:60]: Is it possible to put the card as a component?

**Comment by rwx-yxu** [comment]
The cards have varying it, is it possible to make the heights the same for consistency?

**Comment by advweb-grp1** [comment]
You have also not removed "Title:", "description" ect

**Comment by rwx-yxu** [comment]
The cards do not present themselves as clickable if you hover over them. You can click on them but they open in the same tab. I think it would be easier to have a "Open Article" button rather than make the entire card clickable in this case. Like this:
image

**Comment by AymanReh** [comment]
Final news view 
image
When clicking go to news article opens up a new tab aswell

File: src/views/HomeView.vue (hunk 1 above)
Comment (advweb-grp1) [reply src/views/HomeView.vue]: Not feasible with the time left on the project.

**Pull request merged**